doc/developer-guide/plugins/hooks-and-transactions/ssl-hooks.en.rst

```diff
@@ -84,11 +84,28 @@ handshake processing will not proceed until :c:func:`TSSslVConnReenable()` is ca
 It may be useful to delay the TLS handshake processing if other resources must be consulted to select or create
 a certificate.
 
+TS_VCONN_OUTBOUND_START_HOOK
```

Comment:
TS_VCONN_OUTBOUND_START_HOOK - The use case I need to address requires that the plugin set the certificate used in the origin side TLS mutual auth, based on information that came in on client side request.  This hook looks like it will be perfectly timed.  👍🏻

Reply:
Yes. that was our motivating use case as well.